core/bind/core_bind.cpp

```diff
@@ -1673,8 +1673,14 @@ int _Directory::get_current_drive() {
 }
 
 Error _Directory::change_dir(String p_dir) {
-	ERR_FAIL_COND_V_MSG(!is_open(), ERR_UNCONFIGURED, "Directory must be opened before use.");
-	return d->change_dir(p_dir);
+	Error err = d->change_dir(p_dir);
+	
+	if(err != Error::OK) {
```

Comment:
Scoping is not necessary, consistent with other codebase.
Also clang-format asks to add a space there. 🙂

```suggestion
	if (err != OK) {
```